### DISCUSSION TITLE
Register input in parent component and use it in child component?

### DISCUSSION THREAD

**Discussion by illispi** [react-hook-form/react-hook-form#7325] [Q&A]
I just can't figure out how to do this.

**Comment by bluebill1049** [comment]
provide a simple codesandobx

**Reply by illispi** [reply]
https://codesandbox.io/s/throbbing-cookies-lnty3?file=/src/App.js

**Reply by bluebill1049** [reply]
![Screen Shot 2021-12-21 at 12 37 09 pm](https://user-images.githubusercontent.com/10513364/146855946-2acaf906-3a4d-4b62-b794-6055a56694cf.png)

you never register any input.
